app/views/sensor/readings/_reading.html.haml

```diff
@@ -1,6 +1,13 @@
-%tr.sensor-reading-row{:id => "sensor-reading-" + reading.id.to_s}
-  %td= reading.id
-  %td= reading.created_at
-  %td= reading.calibrated_value
-  %td= reading.uncalibrated_value
-  %td= link_to 'Delete', report_sensor_reading_path(report_id: @report.id, sensor_id: @sensor.id, sensor_reading_id: reading.id), remote: true, method: :delete, :class => "btn btn-secondary"
+%tr.item-table__item.sensor-reading-row{:id => "sensor-reading-#{reading.id}"}
+  %td.align-middle= reading.created_at.strftime(Report::TIME_FORMAT)
+  %td.align-middle= reading.calibrated_value
+  %td.align-middle= reading.uncalibrated_value
+  %td.align-middle.annotation
+    = simple_form_for reading, url: report_sensor_reading_path(@report, @sensor, reading) do |f|
```

Comment:
Maybe outsourcing the annotation form into a separate partial would make this a bit more readable?